README.md

```diff
@@ -79,6 +79,72 @@ Record.fromUint8Array(
     1
   ])
 )
+
+// From a JSON Document
+let json = { hello: 'world' }
+Record.fromJSON(json)
+
+// From a PDF Document
+let pdf = fs.readFileSync('./assets/dummy.pdf')
+Record.fromPDF(pdf)
+```
+
+#### Sign Record
+
+You can sign a Record if it's of type Document: JSON or PDF. In order to sign a Record it's important to provide a private key. We are going to use the private key to encrypt the digital signature. The private key must be stored securely.
```

Comment:
Replace the first part with:

"You can sign a Record if it was constructed from JSON or from a PDF file. To use the sign functionality you need to provide private key as it's going to be used for the digital signature process. Remember to keep it safe!"